jvm/ml4ir-inference/src/main/java/ml4ir/inference/tensorflow/SimpleExampleExecutor.java

```diff
@@ -0,0 +1,33 @@
+package ml4ir.inference.tensorflow;
+
+import com.google.common.collect.ImmutableMap;
+import ml4ir.inference.tensorflow.data.ModelFeaturesConfig;
+import ml4ir.inference.tensorflow.data.StringMapExampleBuilder;
+import org.tensorflow.example.Example;
+import java.util.Map;
+
+public class SimpleExampleExecutor {
```

Comment:
Could we add a javadoc for the class?

Reply:
Out of curiosity, why "Simple"ExampleExecutor?

Reply:
So I'm not sure if "simple" is the best prefix.  More explanatory would be that it's a StringMapExampleExecutor (performs execution of a model taking in a single Map<String, String> and transforming it into an Example for inference.

It's "simple" because having the input be a hashmap of feature-name to feature-value is about as unstructured/simple an input as we can have.